source/shared/cpp/ObjectModel/CollectionTypeElement.h

```diff
@@ -109,8 +109,12 @@ namespace AdaptiveSharedNamespace
 
         collection->SetBleed(ParseUtil::GetBool(value, AdaptiveCardSchemaKey::Bleed, false));
 
-        collection->SetMinHeight(
-            ParseSizeForPixelSize(ParseUtil::GetString(value, AdaptiveCardSchemaKey::MinHeight), &context.warnings));
+        if (const auto& minHeight =
+                ParseSizeForPixelSize(ParseUtil::GetString(value, AdaptiveCardSchemaKey::MinHeight), &context.warnings);
+            minHeight.has_value())
+        {
+            collection->SetMinHeight(minHeight.value());
```

Comment:
can you change it to valuer_or()? iOS doesn't have value()

Reply:
going to try using `operator*` instead of `value()` in case that works on iOS -- see 24dbfdd